### PR TITLE
LGA-feature 2244 standards and cleanup

### PR DESCRIPTION
**Pull request by psweeting1**
Setting agreed standards and new file combinations and restructure. 

For best results in reviewing this PR, please check onto the branch.

### CODE REVIEW

**Comment by jogilder** [comment]
`cla_in_scope.py` should be rolled into `public_eligible.py` and that file (and corresponding feature file) might be better called `public_eligibility` as it covers both eligible and not eligible.

File: README.md [hunk 1]
```diff
@@ -56,3 +56,19 @@ the default.
 ## Docker image
 This repository builds a docker image that is then used as the executor for the `behave` Circle CI job.
 You are unlikely to need to use the image directly elsewhere.
+
```
Comment (jogilder) [inline README.md:59]: Should this be in the other readme for how to run the tests?

File: behave/features/steps/common_steps.py [hunk 2]
```diff
@@ -102,28 +101,7 @@ def compare_client_details_with_backend(context, case_id, client_section):
         ), f"For {title_value}, value displayed is {displayed_value} but actual value is {backend_value}"
 
 
-def step_click_submit(context):
```
Comment (jogilder) [inline behave/features/steps/common_steps.py:105]: what did this get replaced with?

Comment (jogilder) [reply behave/features/steps/common_steps.py:105]: not being used.

File: README.md (hunk 1 above)
Comment (psweeting1) [reply README.md]: I've moved it into the same README.md that contains all the instructions about how to use the tests in docker and run them. Including extra documentation on the commands to run, as I notice we haven't written that down anywhere.

**Comment by psweeting1** [comment]
> `cla_in_scope.py` should be rolled into `public_eligible.py` and that file (and corresponding feature file) might be better called `public_eligibility` as it covers both eligible and not eligible.
> 
> Need to remove empty fixtures folder

Done. As for the empty fixture, my IDE was hiding that random directory `__pycache__` So I never saw it. I had to use visual studios to show it.

**Review by mtac50 — APPROVED** [review]
Looks good!

**Pull request merged**